pandas-stubs/core/frame.pyi

```diff
@@ -207,7 +211,12 @@ class DataFrame:
     def isin(self, values: Union[Iterable, Series, DataFrame, Dict]) -> DataFrame: ...
     def isna(self) -> DataFrame: ...
     def isnull(self) -> DataFrame: ...
-    def iterrows(self) -> Generator[Series, Series, Series]: ...
+    def iterrows(self) -> Iterator[Tuple[_Label, Series]]: ...
+    def itertuples(
+        self, index: bool = ..., name: Optional[_str] = ...
+    ) -> Iterator[
+        Tuple[Any, ...]
+    ]: ...  # Don't know how to express a named tuple with a variable name/num of slots
```

Comment:
What happens if you return `NamedTuple` (from `typing`)? Does mypy complain that it doesn't have any entries?

Reply:
When I tried `NamedTuple`, Mypy infers `Any` as the type of the expression `next(df.itertuples(name='some name'))`.